src/v2/view-builder/views/idp/ChallengeAuthenticatorIdPBody.js

```diff
@@ -0,0 +1,23 @@
+import { loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+
+const Body = BaseForm.extend({
+
+  className: 'challenge-idp',
```

Comment:
learned from Haisheng, we don't need classnames here, instead we should use the ones we have in a `div` a couple of levels above. BaseView is taking care of creating those classnames for you based on the authenticator key